### ISSUE TITLE
PDFUA-221: Figure without visible text, but with Alt text and ActualText | Fail

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-7#221]
Jira issue originally created by user @MarkusErle:

This example demonstrates the failure condition that applies when an image has both [ActualText](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#actualtext) and [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text) set. Only the use of [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text), however, is appropriate in this example because the image conveys information and neither contains essential visible text nor represents essential text.

**Comment by zakkinsey** [comment]
@MarkusErle added a link to #178: This issue clones #178

**Comment by zakkinsey** [comment]
@MarkusErle added a link to #222: This issue is cloned by #222

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

Alt text and ActualText should be different from each other in examples that contain both. 

Alt = sunflower in a field

**Comment by zakkinsey** [comment]
@zakkinsey unassigned issue from @MarkusErle as part of jira-&gt;github migration
